client/my-sites/controller.js

```diff
@@ -238,7 +233,7 @@ function onSelectedSiteAvailable( context ) {
 				.slice( 0, 5 )
 				.filter( recentId => {
 					const site = getSite( state, recentId );
-					return site && site.capabilities;
+					return !! site;
```

Comment:
we can remove the function here, for a simpler `.filter( recentId => ! isNull( getSite( state, recentId ) ) );`

Reply:
Good point that we don't need a variable here. We're okay here with a simple truthy check, but I added the `!!` to note that we are interested in a boolean vs, trying to map over this. Shortened this in b0ebdfb